app/org/sagebionetworks/bridge/play/controllers/BaseController.java

```diff
@@ -42,6 +43,8 @@
 import org.sagebionetworks.bridge.services.StudyService;
 
 import org.apache.commons.lang3.StringUtils;
+import org.codehaus.jackson.JsonGenerationException;
+import org.codehaus.jackson.map.JsonMappingException;
```

Comment:
These are Jackson v1 exceptions. Are they compatible with Jackson v2? We should probably use Jackson v2 for consistency.

Reply:
changed